yarn.lock

```diff
@@ -1404,6 +1404,17 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/react-compose@", "@fluentui/react-compose@^0.19.12":
```

Comment:
Something went wrong there, please check that dependencies are in sync

Reply:
Fixed, it happens when the package generation does not assign dependency versions... and leaves the field as `""`

Reply:
Good catch, I fixed the root cause in #16831